packages/server/src/question/question.controller.ts

```diff
@@ -132,6 +132,24 @@ export class QuestionController {
       isOnline: true,
     }).save();
 
+    if (
```

Comment:
We should probably do this inside question subscriber, and use `QuestionModel.openInQueue` to get the list of open questions (faster and consistent with rest of app)